src/dvclive/fastai.py

```diff
@@ -12,14 +12,29 @@ def __init__(
         model_file: Optional[str] = None,
         with_opt: bool = False,
         live: Optional[Live] = None,
-        **kwargs
+        **kwargs,
     ):
         super().__init__()
         self.model_file = model_file
         self.with_opt = with_opt
         self.live = live if live is not None else Live(**kwargs)
 
+    def before_fit(self):
+        if hasattr(self, "lr_finder") or hasattr(self, "gather_preds"):
+            return
+        params = {
+            "model": type(self.learn.model).__qualname__,
+            "batch_size": getattr(self.dls, "bs", None),
+            "batch_per_epoch": len(getattr(self.dls, "train", [])),
+            "frozen": bool(getattr(self.opt, "frozen_idx", -1)),
+            "frozen_idx": getattr(self.opt, "frozen_idx", -1),
+            "transforms": f"{getattr(self.dls, 'tfms', None)}",
```

Comment:
@alex000kim Asked for your review since I know you use fastai. 

Do you think these params are relevant for being "autologged"? Some missing?

Reply:
I'd add `n_epoch` and `base_lr`.
In general, you can log optimizer's hyperparams as done in wandb and comet integrations:
- https://github.com/fastai/fastai/blob/master/fastai/callback/wandb.py#L117
- https://github.com/fastai/fastai/blob/master/fastai/callback/comet.py#L61